ipfs-api/Cargo.toml

```diff
@@ -28,5 +28,19 @@ ipfs-api-backend-hyper    = { version = "0.5", path = "../ipfs-api-backend-hyper
 
 [dev-dependencies]
 actix-rt                  = "2.5"
+cfg-if                    = "1"
```

Comment:
>  Placement of tests
>
> I noticed that the ipfs_api crate is marked deprecated, but its also not typical to put integration tests in with examples. In this case, it might make sense to do so depending on the future of ipfs_api. Do you prefer the tests in the current location, or in ipfs-api-examples?

I think I prefer this in `ipfs-api-examples` or, to your point, maybe a new crate, `ipfs-api-integration-test`, could make more sense.